automation/run-tests.sh

```diff
@@ -263,6 +263,13 @@ function upgrade_nm_from_copr {
     exec_cmd "systemctl restart NetworkManager"
 }
 
+function run_customize_command {
```

Comment:
Mentioning "pip" in the commit message seems to be unmotivated since the code change does not involve pip at all. Also CI gating is not directly related to this change. I would motivate it as:

Using a customize command is useful when running tests outside a container, too. Therefore always run it when specified. This is for example useful when installing unpackaged dependencies in CI based gating.

Btw. I assumed the gating platforms provide their own way of preparing the test machine for running the test and installing dependencies, so it is probably not hard requirement to have this in there. Nevertheless it seems to be useful to allow this both in containers and non-containers (but this is not yet, what the code seems to do).

Reply:
Modified, thanks!